Doc updates for lookback config feature (#7551)
segmentio/segment-docs: src/engage/audiences/index.md

2. Configure and preview your Audience or Trait.
- A lightning bolt next to `Realtime Enabled` indicates that the computation updates in real-time.

> - You can configure the Include Historical Event Data option to limit how far back event data is processed by setting a lookback window (e.g., “last 90 days”). Unchecking Include Historical Event Data computes values without historical event data, using only data arriving after audience creation.

Reviewer: ```suggestion
- Configure the **Include Historical Event Data** option to limit how far back event data is processed by setting a lookback window (for example, the “last 90 days”). Unchecking **Include Historical Event Data** computes values without historical event data, using only data arriving after audience creation.
```

This error occurs when creating audiences that reference each other, meaning audience X refers to audience Y in its trigger condition, and later you attempt to modify audience Y's trigger condition to refer back to audience X. To avoid this error, ensure that the audiences do not reference each other in their conditions.

### How does the historical data flag work?

> The Include Historical Event Data option lets you take past event data into account and control how much of it is considered when creating real-time audiences. You can set a lookback window (e.g., “last 90 days”) to limit the event data processed, or disable it entirely to use only data arriving after creation. For batch audiences, Segment always includes historical data by default.

Reviewer: ```suggestion
The **Include Historical Event Data** option lets you take past event data into account and control how much of it is considered when creating real-time audiences. You can set a lookback window (for example, the “last 90 days”) to limit the processed event data, or disable it entirely to use only data arriving after creation. For batch audiences, Segment includes historical data by default.
```
